Node/src/botbuilder-teams.d.ts

```diff
@@ -1036,6 +1044,18 @@ export class TeamsChatConnector extends builder.ChatConnector {
   *  @param handler The function to execute when an invoke request to submit task module results is received.
   */
   public onTaskModuleSubmit(handler: TaskModuleSubmitHandlerType): void;
+
+  /**
+  *  Set a handler that is called when an invoke request from command actions to fetch task is received .
+  *  @param handler The function to execute when an invoke request from command actions to fetch task  is received.
+  */
+  public onComposeExtensionFetchTask(handler: ComposeExtensionFetchTaskHandlerType): void;
```

Comment:
Should this be onComposeExtensionFetchTaskInfo?
The notion of "fetching a task" sounds strange to me, especially in the context of a compose extension action.

Reply:
fetching a task actually means fetching a task module. I want to be consistent with the invoke name "composeExtension/fetchTask"